core/src/main/java/org/elasticsearch/cluster/routing/allocation/decider/AwarenessAllocationDecider.java

```diff
@@ -19,21 +19,20 @@
 
 package org.elasticsearch.cluster.routing.allocation.decider;
 
+import java.util.HashMap;
+import java.util.Map;
```

Comment:
> Is this documented somewhere? I have intellij set to use the same order as eclipse defaults.

I don't think it's documented anywhere, it's mostly been that way for a really long time (which isn't an excuse, we should document it). We always used to use Intellij's defaults, where were to put the `java.*` imports below ES imports, I think Eclipse's defaults are the other way around (the way this file is). It doesn't matter which way but it'd be nice to standardize only so we don't have them flipped around for every PR.

Personally I like the original Intellij style, but that's because I already have import-helpers configured to add them in that manner. It also means not switching them around for every PR since 90% of the codebase already uses this format.